### PR TITLE
Cleanup + add more items

### PR DESCRIPTION
**Pull request by bratseth**
@gjoranv or @kkraune or @frodelu please review

### CODE REVIEW

File: documentation/vespa7-release-notes.html [hunk 1]
```diff
@@ -68,10 +112,12 @@ <h3 id="services-xml">services.xml</h3>
   </thead>
   <tbody>
     <tr>
+      <!-- TODO -->
```
Comment (gjoranv) [inline documentation/vespa7-release-notes.html]: What does TODO mean here (and many other places)?

Comment (bratseth) [reply documentation/vespa7-release-notes.html]: When the item is not done/not known by me to be done. To make doubly sure we don't release Vespa 7 with any of these items not done.

Comment (gjoranv) [reply documentation/vespa7-release-notes.html]: All the stuff I added has been done on 7, except for the document/v1 stuff that I commented out.

Comment (gjoranv) [reply documentation/vespa7-release-notes.html]: The protocol is to only add stuff that has been done. Everything else should be on the Vespa 7 theme page.

Comment (bratseth) [reply documentation/vespa7-release-notes.html]: Thanks, removed the TODOs on metrics, rotationscheme and container.filter

Comment (gjoranv) [reply documentation/vespa7-release-notes.html]: Yes, I see.

**Pull request merged**